tools/lur-command.man

```diff
@@ -0,0 +1,49 @@
+.TH lur\-command 1 "Oct 26, 2016" lur\-command
```

Comment:
I’ve adjusted the lur-command manpage and added it to the build. I think the ratbag-command manpage is OK there; I could add ratbag-command again at the end but that doesn’t add much information (perhaps “libratbag” would be better as the “source manual” entry...).